eg01-eg23/eg01-eg23-cycamore.xml

```diff
@@ -1,279 +1,303 @@
   </facility>
 
   <facility>
+    <name>enrichment</name>
     <config>
-      <Source>
-        <capacity>20000000</capacity>
-        <commod>DU2</commod>
-        <recipe_name>DU</recipe_name>
-      </Source>
+      <Enrichment>
+        <feed_commod>natl_u</feed_commod>
+        <feed_recipe>natl_u_recipe</feed_recipe>
+        <product_commod>uox</product_commod>
+        <tails_assay>0.003</tails_assay>
+        <tails_commod>tailings</tails_commod>
+        <swu_capacity>1e100</swu_capacity>
+        <initial_feed>1e100</initial_feed>
+      </Enrichment>
     </config>
-    <name>DU2</name>
   </facility>
 
-
   <facility>
+    <name>sink</name>
     <config>
       <Sink>
-        <capacity>100000000</capacity>
         <in_commods>
-          <val>WASTE</val>
+          <val>cool_uox_waste</val>
+          <val>cool_mox_waste</val>
+          <val>tailings</val>
+          <val>reprocess_waste</val>
+          <val>mox_reprocess_waste</val>
         </in_commods>
-        <max_inv_size>1.000000000000000E+299</max_inv_size>
+        <capacity>1e299</capacity>
       </Sink>
     </config>
-    <name>SINK</name>
-  </facility>
-
-
-  <facility>
-    <config>
-      <FuelFab>
-        <fill_commods>
-          <val>U238</val>
-        </fill_commods>
-        <fill_recipe>U238</fill_recipe>
-        <fill_size>1e+60</fill_size>
-        <fiss_commods>
-          <val>U235</val>
-        </fiss_commods>
-        <fiss_size>1e+60</fiss_size>
-        <outcommod>LWR Fuel</outcommod>
-        <spectrum>thermal</spectrum>
-        <throughput>1e+60</throughput>
-      </FuelFab>
-    </config>
-    <name>LWR Fuel Fab</name>
-  </facility>
-
-
-  <facility>
-    <config>
-      <FuelFab>
-        <fill_commods>
-          <val>DU2</val>
-        </fill_commods>
-        <fill_recipe>DU</fill_recipe>
-        <fill_size>1e+60</fill_size>
-        <fiss_commod_prefs>
-          <val>1</val>
-          <val>2</val>
-        </fiss_commod_prefs>
-        <fiss_commods>
-          <val>LWR Reprocessed</val>
-          <val>FR Reprocessed</val>
-        </fiss_commods>
-        <fiss_size>1e+60</fiss_size>
-        <outcommod>FR Fuel</outcommod>
-        <spectrum>fission_spectrum_ave</spectrum>
-        <throughput>1e+60</throughput>
-      </FuelFab>
-    </config>
-    <name>FR Fuel Fab</name>
   </facility>
 
-
   <facility>
+    <name>uox_reprocessing</name>
     <config>
       <Separations>
-        <feed_commod_prefs>
-          <val>2</val>
-        </feed_commod_prefs>
-        <feed_commods>
-          <val>LWR Spent Fuel</val>
-        </feed_commods>
-        <feedbuf_size>1e+100</feedbuf_size>
-        <leftover_commod>WASTE</leftover_commod>
-        <streams>
+         <feed_commods>   <val>cool_uox_waste</val> </feed_commods>
+         <feed_commod_prefs> <val>1.0</val> </feed_commod_prefs>
+         <feed_recipe>uox_used_fuel_recipe</feed_recipe>
+         <feedbuf_size>1E100</feedbuf_size>
+         <throughput>1E100</throughput>
```

Comment:
- What would you consider a realistic number for this kind of scenario?
much less than 1E100

- "I was thinking that a realistic approximation can be set after running a successful simulation (when the recipes are set and all,). "
Yes, that is what we discussed in my office a few weeks ago, and of course I agree. Has this input file never been run?